method/scanpy/config/config_default.json

```diff
@@ -0,0 +1,5 @@
+{ 
+    "n_neighbors" : 15, 
+    "clustering" : "leiden",
+    "source": "https://scanpy.readthedocs.io/en/stable/generated/scanpy.pp.pca.html,https://scanpy.readthedocs.io/en/stable/api/generated/scanpy.pp.neighbors.html, https://scanpy.readthedocs.io/en/stable/generated/scanpy.pp.highly_variable_genes.html"
+}
```

Comment:
Shouldn't the default also have the `directed` and `n_iterations` parameters?

Reply:
The default settings for these parameters are within the script, lines 192-193:

```
directed = None if "directed" not in config.keys() else config["directed"]
n_iterations = -1 if "n_iterations" not in config.keys() else config["n_iterations"]
```
Would you prefer to put them in the default config instead?

Reply:
I would personally prefer it to be explicit in the config instead of the code. But that's just me :)
I'll approve, and you can change it if you feel like or merge if not :)